src/protocol/i3bar_event.rs

```diff
@@ -37,7 +37,7 @@ fn unprocessed_events_stream(invert_scrolling: bool) -> BoxedStream<I3BarEvent>
             #[derive(Deserialize)]
             struct I3BarEventRaw {
                 instance: Option<String>,
-                button: MouseButton,
+                button: i64,
```

Comment:
What's the benefit of using an i64 instead of keeping it as a `MouseButton`?

Reply:
I wanted to handle the case in which bar sends a click event with button we don't support / know about. Now I see that is it better to leave this as a `MouseButton` and gracefully handle deserialization errors instead of unwrapping.